File: src/main/java/commands/misc/UserInfo.java
package commands.misc;

import com.jagrosh.jdautilities.doc.standard.CommandInfo;
import commandHandler.Command;
import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
import org.jetbrains.annotations.NotNull;

import java.time.OffsetDateTime;
import java.time.format.DateTimeFormatter;
import java.util.ArrayList;

@CommandInfo(name = "userinfo", description = "get your users info", usage = "userinfo")
public class UserInfo extends Command {

    @Override
    protected void execute(@NotNull MessageReceivedEvent event, ArrayList<String> args) {
        EmbedBuilder eb = new EmbedBuilder();
        OffsetDateTime time = event.getMember().getTimeBoosted();
        String nick = event.getMember().getNickname();

        eb
                .setTitle(event.getAuthor().getName() + "'s Info")
                .setThumbnail(event.getAuthor().getAvatarUrl())
                .setDescription("**ID: `" + event.getAuthor().getId() + "`**\n" +
                        "        **Discrim:** #" + event.getAuthor().getDiscriminator() + "\n" +
                        "        **Guild Nickname:** " + (nick != null ? nick : "no nick") + "\n" +
                        "        **Joined Server:** " + event.getMember().getTimeJoined().format(DateTimeFormatter.ofPattern("dd-MM-yyyy")) + "\n" +
                        "        **Avatar URL:** [Click](" + event.getAuthor().getAvatarUrl() + ")\n" +
                        "        **Started Boosting: **" + (time != null ? time.format(DateTimeFormatter.ofPattern("dd-MM-yyyy")) : "not boosted yet"));
        event.getChannel().sendMessage(eb.build()).queue();
    }
}

File: src/main/kotlin/commands/misc/UserInfo.kt
package commands.misc

import com.jagrosh.jdautilities.commons.waiter.EventWaiter
import com.jagrosh.jdautilities.doc.standard.CommandInfo
import commandHandler.Command
import net.dv8tion.jda.api.EmbedBuilder
import net.dv8tion.jda.api.events.message.MessageReceivedEvent
import java.time.OffsetDateTime
import java.time.format.DateTimeFormatter
import java.util.*

@CommandInfo(name = ["userinfo"], description = "get your users info", usage = "userinfo")
class UserInfo constructor(waiter: EventWaiter?) : Command(waiter) {
    override fun execute(event: MessageReceivedEvent, args: ArrayList<String>) {
        val eb = EmbedBuilder()
        val time: OffsetDateTime? = event.member!!.timeBoosted
        eb
                .setTitle(event.author.name + "'s Info")
                .setThumbnail(event.author.avatarUrl)
                .setDescription(("**ID: `" + event.author.id + "`**\n" +
                        "        **Discrim:** #" + event.author.discriminator + "\n" +
                        "        **Guild Nickname:** " + (event.member!!.nickname ?: "no nick") + "\n" +
                        "        **Joined Server:** " + event.member!!.timeJoined.format(DateTimeFormatter.ofPattern("dd-MM-yyyy")) + "\n" +
                        "        **Avatar URL:** [Click](" + event.author.avatarUrl + ")\n" +
                        "        **Started Boosting: **" + (if (time != null) time.format(DateTimeFormatter.ofPattern("dd-MM-yyyy")) else "not boosted yet")))
        event.channel.sendMessage(eb.build()).queue()
    }
}